backend/management/commands/ingest_orcid.py

```diff
@@ -0,0 +1,13 @@
+from django.core.management.base import BaseCommand
```

Comment:
Same should be done for BASE import, see https://github.com/dissemin/dissemin/pull/760#discussion_r284234579.